express_server.js

```diff
@@ -6,135 +6,175 @@ const cookieSession = require("cookie-session");
 const bcrypt = require("bcrypt");
 const methodOverride = require("method-override");
 // helper functions:
-const { generateID, getUserByEmail, userURLs, copy } = require("./helpers.js");
+const {
+  areValidEmailPassword,
+  generateID,
+  getUserByEmail,
+  hashPassword,
+  userURLs,
+  emailExists,
+} = require("./helpers.js");
 
 app.set("view engine", "ejs");
-app.use(bodyParser.urlencoded({extended: true}));
-app.use(cookieSession({
-  name: "session",
-  keys: ["user_id"]
-}));
-app.use(methodOverride('_method'));
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(
+  cookieSession({
+    name: "session",
+    keys: ["user_id"],
+  }),
+);
+app.use(methodOverride("_method"));
```

Comment:
Some changes are only because of "prettying" the code. If you haven't heard or used Prettier or other tools like that, look it up. They help you format your code by typing a quick shortcut or they do it automatically after saving the file.